op.c

```diff
@@ -16171,14 +16171,22 @@ Perl_coresub_op(pTHX_ SV * const coreargssv, const int code,
     PERL_ARGS_ASSERT_CORESUB_OP;
 
     switch(opnum) {
-    case 0:
+    case 0: {
+        int caller_index = -1;
```

Comment:
This is just for passing into `newSViv`, so why not:
```suggestion
        IV caller_index = -1;
```

Reply:
I don't have such deep knowledge yet, old code used `int % 3` pattern so I just used it as it was

Reply:
945442cc8d